hedy.py

```diff
@@ -1621,7 +1621,12 @@ def transpile_inner(input_string, level):
             raise exceptions.UnsupportedFloatException(value=''.join(invalid_info.arguments))
         else:
             invalid_command = invalid_info.command
-            closest = closest_command(invalid_command, commands_per_level[level])
+            closestmatch = closest_command(invalid_command, commands_per_level[level])
+            closest = style_closest_command(closestmatch)
+            if closestmatch:
+                fixed_code = input_string.replace(invalid_command, closestmatch)
+                if fixed_code != input_string: #only if we have made a successful fix
+                    result = transpile_inner(fixed_code, level)
```

Comment:
If the "repaired" code has a problem too, the problem for the "repaired" code will be thrown as exception by this `transpile_inner`. You will need to catch it so you can still report the original problem.

Reply:
yes you would want to check not only if fixed_code is not the same as the original code, but also if it is valid code (but that is a program repair issue while this PR is mainly a UX PR, so we can fix that separately)